Migrate to a new AmneziaWG Docker Image

AlexanderGalkov opened the pull request:
Update server scripts

Commit:
rename interface and config file name to awg0
change base docker image to amneziavpn/amneziawg-go:latest

pokamest commented:
Probably need to provide backward compatibility with older container version, look ClientManagementModel::getWireGuardClients

Nethius committed:
Merge branch 'dev' of github.com:amnezia-vpn/amnezia-client into update_server_scripts

Nethius committed:
updated the paths to awg files and interfaces to match the new docker container

Nethius committed:
Merge branch 'dev' of github.com:amnezia-vpn/amnezia-client into feature/new-awg-docker-container

Nethius committed:
added backward compatibility for the old awg container

Nethius committed:
Merge branch 'dev' of github.com:amnezia-vpn/amnezia-client into feature/new-awg-docker-container

Nethius committed:
Merge branch 'dev' of github.com:amnezia-vpn/amnezia-client into HEAD

Nethius committed:
Merge branch 'update_server_scripts' of github.com:amnezia-vpn/amnezia-client into feature/new-awg-docker-container

pokamest committed:
Merge pull request #628 from amnezia-vpn/feature/new-awg-docker-container

Feature/new awg docker container

pokamest committed:
Merge branch 'dev' into update_server_scripts

pokamest committed:
Merge branch 'dev' into update_server_scripts

Nethius committed:
Merge branch 'dev' of github.com:amnezia-vpn/amnezia-client into HEAD

Nethius committed:
Merge branch 'dev' of github.com:amnezia-vpn/amnezia-client into HEAD

lunardunno committed:
Linux version to stdOut

Print the Linux version to stdOut for subsequent checking by the server controller.

lunardunno committed:
revert change: Linux versions of stdOut
